test/blackbox-tests/test-cases/describe/describe.t

```diff
@@ -1316,33 +1316,34 @@ Test other formats
 Test errors
 -----------
 
-  $ dune describe --lang 0.1 workspac
-  Error: Unknown constructor workspac
-  Hint: did you mean workspace?
+  $ dune describe workspacw --lang 0.1 
```

Comment:
This test would actually succeed since `cmdliner` can infer that `workspac` -> `workspace`. Therefore I changed it to actually be incorrect.